dynawo/sources/Common/test/TestIoDico.cpp

```diff
@@ -32,14 +33,14 @@ TEST(CommonIoDicoTest, testCommonIoDicosTest) {
   ASSERT_EQ(IoDicos::hasIoDico("MyIoDico"), false);
   ASSERT_THROW(IoDicos::getIoDico("MyIoDico"), MessageError);
 
-  boost::shared_ptr<IoDicos> dicos = IoDicos::getInstance();
-  dicos->addPath("res");
-  ASSERT_NO_THROW(dicos->addDico("MyIoDico", "dico", ""));
-  ASSERT_THROW(dicos->addDico("MyIoDico", "", ""), MessageError);
-  ASSERT_THROW(dicos->addDico("MyIoDico", "MyDummyDico", ""), MessageError);
+  IoDicos& dicos = IoDicos::getInstance();
```

Comment:
Is there a way to unit test that the Trace and Dicos are always unique? Maybe by compiling and loading a dummy library that writes something in the log and make sure it ends up in the proper file? Might be the subject of another commit though.